frontends/precinct-scanner/src/app_root.tsx

```diff
@@ -643,14 +316,55 @@ export function AppRoot({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isPollsOpen, initializedFromStorage]);
 
-  function dismissError() {
-    /* istanbul ignore next */
-    if (timeoutToInsertScreen) {
-      window.clearTimeout(timeoutToInsertScreen);
+  const togglePollsOpen = useCallback(() => {
+    dispatchAppState({ type: 'togglePollsOpen' });
+  }, []);
+
+  const toggleTestMode = useCallback(async () => {
```

Comment:
TODO: move away from `toggle` APIs and have them be idempotent, i.e. `setTestMode(true/false)`.

Reply:
Yes I agree, toggle scares me. Was trying to minimize my changes when possible so resisted changing this one